database/insertData.js

```diff
@@ -1542,6 +1542,16 @@ const queries = [
                 sequence: 2
                 parameterQueries: { message: "Application Submitted" }
               }
+              {
+                actionCode: "changeStatus"
+                trigger: ON_REVIEW_CREATE
+                parameterQueries: {
+                  applicationId: {
+                    operator: "objectProperties"
+                    children: [{ value: { property: "reviewId" } }]
+                  }
+                  newStatus: { value: "Draft" }
+              }
```

Comment:
```suggestion
            {
              actionCode: "changeStatus"
              trigger: ON_REVIEW_CREATE
              parameterQueries: {
                applicationId: {
                  operator: "objectProperties"
                  children: [{ value: { property: "reviewId" } }]
                }
                newStatus: { value: "Draft" }
              }
            }
```
Missing one closing `}`

Reply:
This is the object collected by the Trigger processor that is available to evaluate parameters from:
```
ApplicationData {
   trigger_id: 1,
   trigger: 'onReviewCreate',
   table: 'review',
   record_id: 1,
   applicationId: 4,
   templateId: 4,
   stageId: 5,
   stageNumber: 1,
   stage: 'Screening',
   stageHistoryId: 3,
   stageHistoryTimeCreated: null,
   statusHistoryId: 3,
   status: 'Submitted',
   statusHistoryTimeCreated: 2020-12-08T23:42:34.307Z,
   userId: 5,
   firstName: 'John',
   lastName: 'Smith',
   username: 'js',
   dateOfBirth: null,
   email: 'js@nowhere.com',
   responses: {
     PB1: null,
     PB2: { text: 'Turning orange' },
     Q1: null,
     Q2: { text: '39' },
     Q20: { text: '50mg' },
     Q21: { text: '100' },
     Q3: { text: 'New Zealand' },
     Q4: { text: 'Vitamin C' },
     Q5: { text: 'Natural Product', optionIndex: 1 },
     Text1: { text: 'js@nowhere.com' },
     Text2: null
   }
 }
```

Reply:
I complete stuffed that up, thanks for the suggested fix